jabref-template/listrefs.book.layout

```diff
@@ -0,0 +1,99 @@
+\begingroup{year}
+  <tr class="show"><td><h3>\format[HTMLChars]{\year}</td></tr></h3>
+\endgroup{year}
+
+<tr id="\format{\bibtexkey}" class="entry">
+  <td>
+    \format[Authors(FirstFirst,Initials,Comma,Comma),HTMLChars]{\author}
+    <br>
+    <b>
+      \format[HTMLChars]{\title}
+    </b>
+
+    <br>
+
+   <!-- The following parts of each entry are all optional and may not
+        be present in all bibtex entries that are processed via the
+        current file. If present, the `\begin{field} ... \end{field}`
+        section will be expanded into something nonzero, and we will
+        do this with all of the following pieces of information about
+        each publication. Because we expect the first entry to always
+        be there, we add all following entries with a leading
+        comma. To avoid a space between the previous and current entry
+        (which starts with a comma), we end end each line with a
+        comment-start marker and begin the next line with the
+        comment-end marker. 
+
+        Everything is closed with a period after the last entry.
+
+        -->
+
+    -->\begin{publisher}\format[HTMLChars]{\publisher}\end{publisher}<!--
```

Comment:
Thanks for adding this @bangerth! I've had a look at the CI `output.html`, and there's an errant `-->` that appears before the publishers name. 
```
T. Wick
Multiphysics Phase-Field Fracture: Modeling, Adaptive Discretizations, and Solvers
-->De Gruyter, Berlin, Boston, 2020. 
```
Maybe it's the one at the beginning of this line?

Reply:
Ah, good catch. Can you try again?

Reply:
Fixed! Looks great :-)